api-specs/api/resources/in-store.raml

```diff
@@ -1061,3 +1061,186 @@ uriParameters:
             application/json:
               type: ShippingMethodPagedQueryResponse
               example: !include ../examples/shipping-methods.example.json
+/shopping-lists:
+  type:
+    baseDomain:
+      resourceType: ShoppingList
+      resourceQueryType: ShoppingListPagedQueryResponse
+      resourceDraft: ShoppingListDraft
+      whereExample: key = "my-shopping-list"
+      sortExample: createdAt asc
+  description: |
+    shopping-lists e.g. for wishlist support
+  get:
+    (java-implements):
+      'com.commercetools.api.models.PagedQueryResourceRequest<ByProjectKeyShoppingListsGet,
```

Comment:
This is the wrong type in the generic. Please look it up in the java sdk. Should be something like `ByProjectKeyInstoreKeyByKeyValueShoppingListGet`